root/var/www/html/freepbx/rest/modules/configuration.php

```diff
@@ -141,3 +141,27 @@ function setLegacyMode($value) {
     }
 });
 
+$app->get('/configuration/ldapphonebook', function (Request $request, Response $response, $args) {
+    try {
+        $configuration = array();
+        exec("/usr/bin/sudo /sbin/e-smith/config getjson phonebookjs", $out);
+        $tmp = json_decode($out[0]);
+        $configuration['ldap'] = array();
+        $configuration['ldap']['enabled'] = ($tmp->props->status == 'enabled') ? true : false;
+        $configuration['ldap']['port'] = $tmp->props->TCPPort;
```

Comment:
Always return the same object schema

```suggestion
        $configuration['ldap']['port'] = $tmp->props->TCPPort;
        $configuration['ldap']['user'] = '';
        $configuration['ldap']['password'] = '';
```